api/proxy/servers/arbitrum_altda/cli.go

```diff
@@ -0,0 +1,51 @@
+package arbitrum_altda
+
+import (
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	ListenAddrFlagName = "arbitrum-da.addr"
+	PortFlagName       = "arbitrum-da.port"
+	Enabled            = "arbitrum-da.enabled"
+)
+
+func withEnvPrefix(prefix, s string) []string {
+	return []string{prefix + "_ARB_CUSTOM_DA_" + s}
```

Comment:
why CUSTOM? Why not just `_ARBITRUM_DA_` to match the flag names?

Reply:
https://github.com/Layr-Labs/eigenda/pull/1957/commits/7cccd3cb6b51137f26677e1ce1e085cc115f9f50